public/app/features/plugins/sql/datasource/SqlDatasource.ts

```diff
@@ -146,7 +146,7 @@ export abstract class SqlDatasource extends DataSourceWithBackend<SQLQuery, SQLO
     return lastValueFrom(
       getBackendSrv()
         .fetch<BackendDataSourceResponse>({
-          url: '/api/ds/query',
+          url: '/api/ds/query?type=sql', // ?? which SQL!
```

Comment:
Isn't `this.type` available here or from `super.type`?

Reply:
🤦